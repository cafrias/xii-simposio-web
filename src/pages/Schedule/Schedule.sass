@import '_sass/_breakpoints.sass'
@import '_sass/_constants.sass'

.mobile-content
  width: 100%
  padding: 1.618rem

.desktop-content
  display: none
  width: 80%
  max-width: 1400px
  margin: 2.618rem 0

// TABLE _______________________________________________________________________
.schedule-table
  width: 100%
  border-collapse: collapse

.mobile-table
  margin-bottom: 2.618rem

.desktop-table
  width: 100%

.table__date
  font-size: 2.023rem
  padding: 1rem 0
  color: $secondary
  text-align: center !important

.table__weekday
  display: block
  color: $dark-brown
  font-size: 1.618rem

.activity-row>td
  border-top: 1px solid $light-brown
  padding: 1rem 0

.activity__name
  text-align: center

.activity__time
  font-weight: 900
  color: $dark-brown

// DESKTOP BREAKPOINT __________________________________________________________
+for-desktop-up
  .mobile-content
    display: none

  .desktop-content
    display: block
