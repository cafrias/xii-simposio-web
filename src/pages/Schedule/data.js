export const timeEntries = [
  '15.00',
  '17.00',
  '17.30',
  '19.00',
  '19.30',
  '21.00',
  '22.30'
]

export default [
  {
    weekday: 'Miércoles',
    date: '17/05/2017',
    activities: {
      '15.00': 'Acreditaciones',
      '17.00': 'Acto de Apertura',
      '17.30': 'Trabajo en comisiones',
      '19.00': 'Coffee Break',
      '19.30': 'Conferencia',
      '21.00': 'Cóctel de bienvenida'
    }
  },
  {
    weekday: 'Jueves',
    date: '18/05/2017',
    activities: {
      '17.00': 'Trabajo en comisiones',
      '17.30': 'Trabajo en comisiones',
      '19.00': 'Coffee Break',
      '19.30': 'Conferencia',
      '21.00': 'Cena libre'
    }
  },
  {
    weekday: 'Viernes',
    date: '19/05/2017',
    activities: {
      '17.00': 'Trabajo en comisiones',
      '17.30': 'Trabajo en comisiones',
      '19.00': 'Coffee Break',
      '19.30': 'Conferencia',
      '21.00': 'Acto de clausura',
      '22.30': 'Cena de camaradería'
    }
  },
]
