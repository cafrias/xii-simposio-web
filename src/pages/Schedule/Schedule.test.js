import React from 'react'
import { mount, shallow } from 'enzyme'

import Schedule from './index'

it('should render without errors', () => {
  const wrapper = mount(<Schedule/>)
  expect(wrapper).toHaveLength(1)
})

it('should match snapshot', () => {
  const wrapper = shallow(<Schedule/>)
  expect(wrapper).toMatchSnapshot()
})
