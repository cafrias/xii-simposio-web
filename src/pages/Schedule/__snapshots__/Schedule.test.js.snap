// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot 1`] = `
<Section
  className="background_white schedule"
  id="schedule"
  name="Cronograma"
>
  <div
    className="mobile-content text-container"
  >
    <MobileTable
      activities={
        Object {
          "15.00": "Acreditaciones",
          "17.00": "Acto de Apertura",
          "17.30": "Trabajo en comisiones",
          "19.00": "Coffee Break",
          "19.30": "Conferencia",
          "21.00": "Cóctel de bienvenida",
        }
      }
      date="17/05/2017"
      key="0"
      weekday="Miércoles"
    />
    <MobileTable
      activities={
        Object {
          "17.00": "Trabajo en comisiones",
          "17.30": "Trabajo en comisiones",
          "19.00": "Coffee Break",
          "19.30": "Conferencia",
          "21.00": "Cena libre",
        }
      }
      date="18/05/2017"
      key="1"
      weekday="Jueves"
    />
    <MobileTable
      activities={
        Object {
          "17.00": "Trabajo en comisiones",
          "17.30": "Trabajo en comisiones",
          "19.00": "Coffee Break",
          "19.30": "Conferencia",
          "21.00": "Acto de clausura",
          "22.30": "Cena de camaradería",
        }
      }
      date="19/05/2017"
      key="2"
      weekday="Viernes"
    />
  </div>
  <div
    className="desktop-content"
  >
    <DesktopTable
      days={
        Array [
          Object {
            "activities": Object {
              "15.00": "Acreditaciones",
              "17.00": "Acto de Apertura",
              "17.30": "Trabajo en comisiones",
              "19.00": "Coffee Break",
              "19.30": "Conferencia",
              "21.00": "Cóctel de bienvenida",
            },
            "date": "17/05/2017",
            "weekday": "Miércoles",
          },
          Object {
            "activities": Object {
              "17.00": "Trabajo en comisiones",
              "17.30": "Trabajo en comisiones",
              "19.00": "Coffee Break",
              "19.30": "Conferencia",
              "21.00": "Cena libre",
            },
            "date": "18/05/2017",
            "weekday": "Jueves",
          },
          Object {
            "activities": Object {
              "17.00": "Trabajo en comisiones",
              "17.30": "Trabajo en comisiones",
              "19.00": "Coffee Break",
              "19.30": "Conferencia",
              "21.00": "Acto de clausura",
              "22.30": "Cena de camaradería",
            },
            "date": "19/05/2017",
            "weekday": "Viernes",
          },
        ]
      }
    />
  </div>
</Section>
`;
